### ISSUE TITLE
Dynamic learning rate callback

### ISSUE THREAD

**Issue by emremidilli** [emremidilli/Time-Series-Forecasting#10]
In case of significant drop of metrics during training, make sure that learning rate is reduced immidately. Model and optimizers are loaded back from the previous checkpoints.
